source/tunneling/SecureTunnelingContext.cpp

```diff
@@ -20,6 +20,8 @@ namespace Aws
             {
                 constexpr char SecureTunnelingContext::TAG[];
 
+                SecureTunnelingContext::SecureTunnelingContext() {}
```

Comment:
Just asking, do we need this here if you define `SecureTunnelingContext()=default` in the header file? Right now SecureTunnelingContext.h and SecureTunnelingFeature.h do not explicitly define the constructor as default in the header, rather they do it in the source file.

Reply:
Oh yeah, that works. Fixed.